internal/compiler/object_tree.rs

```diff
@@ -641,6 +641,7 @@ pub struct ElementDebugInfo {
     // The id qualified with the enclosing component name. Given `foo := Bar {}` this is `EnclosingComponent::foo`
     pub qualified_id: Option<SmolStr>,
     pub type_name: String,
+    pub element_id: u64,
```

Comment:
what's that element_id for?
Could you add a comment describing what it is and what it is for?

Reply:
Comment added in the next push.